packages/contracts/src/hardhat-deploy-ethers.ts

```diff
@@ -4,8 +4,9 @@ import { Provider } from '@ethersproject/abstract-provider'
 import { Signer } from '@ethersproject/abstract-signer'
 import { sleep, awaitCondition } from '@eth-optimism/core-utils'
 import { HttpNetworkConfig } from 'hardhat/types'
+import { cyan, yellow, red } from 'chalk'
```

Comment:
I don't love the idea of including `chalk` as a dep, it adds a point of compromise in a very important set of scripts

Reply:
+1